### PR TITLE
Support ES6 default exports in webpack/cosmos configs

### PR DESCRIPTION
**Pull request by ovidiubute**
See #242

### CODE REVIEW

**Comment by ovidiubute** [comment]
@skidding please review

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 packages/*/lib
 node_modules
 test/coverage
+.idea/
```
Comment (skidding) [inline .gitignore]: What is this? If it's not project-related best put it in your global gitignore.

File: packages/react-cosmos-utils/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cosmos-utils",
-  "version": "2.0.0-beta.1",
+  "version": "2.0.0-beta.2",
```
Comment (skidding) [inline packages/react-cosmos-utils/package.json]: No need to update versions. Lerna takes care of them automagically when running the publish cmd.

File: packages/react-cosmos-webpack/package.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
     "html-webpack-plugin": "^2.24.1",
     "json-loader": "^0.5.4",
     "loader-utils": "^0.2.16",
-    "react-cosmos": "^2.0.0-beta.2",
+    "react-cosmos": "^2.0.0-beta.3",
+    "react-cosmos-utils": "^2.0.0-beta.2",
```
Comment (skidding) [inline packages/react-cosmos-webpack/package.json]: No need to update dep versions. Lerna takes care of them automagically when running the publish cmd.

**Comment by ovidiubute** [comment]
@skidding `.idea` is the project directory for WebStorm. I removed it from `gitignore`, and I also reverted the package versions, didn't realise you were using Lerna. Please review again.

**Pull request merged**